mailer.js

```diff
@@ -26,6 +27,19 @@ module.exports = function (log) {
     return 'href="' + url + '" style="color: #0095dd; text-decoration: none; font-family: sans-serif;"'
   }
 
+  function constructLocalTimeString (timeZone, locale) {
+    // if no timeZone is passed, use 'Etc/UTC'
+    moment.tz.setDefault('Etc/UTC')
+    // if no locale is passed, use 'en'
+    locale = locale || 'en'
+    moment.updateLocale(locale)
+    var time = moment()
+    if (timeZone) {
+      time = time.tz(timeZone)
+    }
+    // return a locale-specific time
+    return time.format('LTS (z) dddd, ll')
```

Comment:
This is only a pseudo-locale-specific time. We are specifying the format that says time must come first, then timezone, then full day of the week, then localized date.

We should consider using `LLLL` or `llll` which will translate all that information into the format best suited for that locale, with the caveat that the result will lack the user's timezone.


## LLLL

<img width="373" alt="screen shot 2016-07-22 at 11 22 47" src="https://cloud.githubusercontent.com/assets/848085/17054367/b06c5d68-4ffe-11e6-89d1-3227eb7f9798.png">

## ll

<img width="379" alt="screen shot 2016-07-22 at 11 22 58" src="https://cloud.githubusercontent.com/assets/848085/17054375/b60ecdaa-4ffe-11e6-80c0-497d2e46c58c.png">

For v1, I'm not going to block on this.